.circleci/config.yml

```diff
@@ -48,7 +51,7 @@ workflows:
           requires:
             - orb-tools/pack
             - shellcheck/check
-          vcs-type: github
+          vcs-type: << pipeline.project.type >>
       - orb-tools/continue:
```

Comment:
Can you add a comment here referencing the test-deploy.yml file and the workflow there?